pkg/daemon/ceph/osd/volume_test.go

```diff
@@ -238,21 +239,57 @@ var cephVolumeTestResultMultiClusterMultiOSD = `{
 }
 `
 
-func TestParseCephVolumeResult(t *testing.T) {
+var cephVolumeRAWTestResult = `{
+    "0": {
+        "ceph_fsid": "4bfe8b72-5e69-4330-b6c0-4d914db8ab89",
+        "device": "/dev/vdb",
+        "osd_id": 0,
+        "osd_uuid": "c03d7353-96e5-4a41-98de-830dfff97d06",
+        "type": "bluestore"
```

Comment:
FWIW, the OSD code no longer obeys `type` and always creates bluestore OSDs. This should be removed from the API spec, but I found this to be one of the most difficult parts of stripping out all the OSD code for legacy OSDs. We might do this in a separate PR.